texecomConnect.py

```diff
@@ -800,7 +826,9 @@ def event_loop(self):
                 self.log("Connection lost")
             connectionLostTime = time.time() - lastConnectedAt
             if connectionLostTime >= 60 and not notifiedConnectionLoss:
-                self.log("Connection lost for over 60 seconds - calling send-message.sh")
+                self.log(
+                    "Connection lost for over 60 seconds - calling send-message.sh"
```

Comment:
IMHO this can be removed (along with the line below it where the script is called and any other references), as the bash script isn't in the repo, and any notifications are now sent via mqtt